pkg/volume/configmap/configmap.go

```diff
@@ -249,17 +249,17 @@ func (b *configMapVolumeMounter) SetUpAt(dir string, mounterArgs volume.MounterA
 		return err
 	}
 
-	err = writer.Write(payload)
+	setPerms := func(tsDir string) error {
```

Comment:
if you are going to ignore the argument, name it `_` which tells Go it can safely optimize and tells readers this was on purpose

Reply:
Fixed